front/src/components/HeaderMenu/HeaderMenuContainer.tsx

```diff
@@ -1,136 +1,93 @@
-import { useContext, useMemo, useState } from "react";
-import { useAuth0 } from "@auth0/auth0-react";
-import HeaderMenu, { MenuItemsGroupData } from "./HeaderMenu";
+import { useContext, useMemo } from "react";
 import { GlobalPreferencesContext } from "providers/GlobalPreferencesProvider";
 import QrReaderOverlay from "components/QrReaderOverlay/QrReaderOverlay";
 import { qrReaderOverlayVar } from "queries/cache";
 import { useReactiveVar } from "@apollo/client";
+import { useAuthorization } from "hooks/useAuthorization";
+import HeaderMenu, { IMenuItemsGroupData } from "./HeaderMenu";
 
-const HeaderMenuContainer = () => {
-  const auth0 = useAuth0();
+function HeaderMenuContainer() {
+  const authorize = useAuthorization();
   const { globalPreferences } = useContext(GlobalPreferencesContext);
   const baseId = globalPreferences.selectedBase?.id!;
   const qrReaderOverlayState = useReactiveVar(qrReaderOverlayVar);
 
-  const menuItems: MenuItemsGroupData[] = useMemo(
-    () => [
+  // TODO: do this at route definition
+  const menuItems: IMenuItemsGroupData[] = useMemo(() => {
+    const oldAppUrlWithBase = `${process.env.REACT_APP_OLD_APP_BASE_URL}/?camp=${baseId}`;
+    return [
       {
-        text: "Classic View",
+        text: "Inventory",
```

Comment:
@aerinsol can you check from here to line 64 if the titles and the required action based permissions are correct?

Reply:
@aerinsol here is the permissions defined that you need to see a menu.